hedera-node/src/main/java/com/hedera/services/state/merkle/MerkleNetworkContext.java

```diff
@@ -264,7 +256,7 @@ public String toString() {
 				.append(stateVersion == UNRECORDED_STATE_VERSION ? "<N/A>" : stateVersion)
 				.append(") is,")
 				.append("\n  Consensus time of last handled transaction :: ")
-				.append(reprOf(consensusTimeOfLastHandledTxn))
+				.append(consensusTimeOfLastHandledTxn)
```

Comment:
The `reprOf` knows what to do with a `null` or a real `RichInstant`. We should keep the call to `reprOf` here, but change the function to receive an `Instant` instead of a `RichInstant`.

Reply:
Yeah, that's right. Changed.